versions/1/manifest.spec.yml

```diff
@@ -295,5 +298,4 @@ spec:
   - title
   - description
   - version
-  - release
```

Comment:
Tested some packages without `release` tag on current versions of Kibana, and they work, but the package appears as experimental.

So I am thinking that we have to keep this as mandatory, to properly display packages in current and older versions, and maybe add semantic validation in the future to make it optional in future versions of kibana where https://github.com/elastic/kibana/issues/122973 is implemented.

Reply:
@jen-huang @joshdover @ruflin would it be an option to change the default in Kibana from "experimental" to "ga", for packages that don't include the `release` field in their manifest? This would need to be done for 7.17.0 and 8.0.0.

This wouldn't change the behaviour for existing packages, because currently the `release` tag is mandatory in the spec. It would only change the behaviour for future packages, that more likely won't support Kibana versions older than 7.17.

Reply:
I was checking what changes would be needed in Kibana for this, and I have found that Kibana takes the release as it comes from the registry. The "experimental" default is there, so we can do the change from our side.

https://github.com/elastic/package-registry/blob/ae68c7ac07afe1180a80a25f7dc2615a68b6b332/packages/releases.go#L13

Reply:
Nice finding :)

Reply:
Can you make sure it is the case also for older release of Kibana? I think it is but just to double check that we don't break 7.14 or similar.

Reply:
> Can you make sure it is the case also for older release of Kibana? I think it is but just to double check that we don't break 7.14 or similar.

Tried today, 7.14 behaves the same.